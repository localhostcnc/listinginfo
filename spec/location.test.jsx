/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import Location from '../dist/Components/lowerInfo/location';

describe('Components', () => {
  test('Rendering the Location component', () => {
    const wrapper = shallow(
      <Location />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
