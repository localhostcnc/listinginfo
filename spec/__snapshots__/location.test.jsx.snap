// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Rendering the Location component 1`] = `ShallowWrapper {}`;
